portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/*
+Shows about text when the "About" button is pressed.
+*/
+function showAbout () {
+    aboutText = "I'm a rising junior at UCLA, majoring in Linguistics and Computer Science (that's one major)! In my free time, I enjoy reading and playing the piano. \n\n Recently, I've started learning to cook (I'm going to need this for my college apartment next year) and playing Stardew Valley (turns out video games are a great way to stay connected to friends during this time)!";
+    var aboutContainer = document.getElementById('aboutSpace');
+    //if about pressed, either remove / show about text depending on what is already there
+    if (aboutContainer.innerText === "") {
+        //change inner text to about text, and div class to CSS about class
+        aboutContainer.innerText = aboutText;
+        aboutContainer.className="background";
+    }
+    else {
+        aboutContainer.innerText = "";
+        aboutContainer.className="";
+    }
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/*
+Shows pictures when the "Pictures" button is pressed.
+*/
+function showPics() {
+    //array of images
+    let imgs = ["ucla1.JPG", "ucla2.jpg.JPG", "book.JPG", "harrypotter.JPG"]
+    //random int from 0 - 4
+    var randInt = (Math.floor(Math.random() * 5));
```

Comment:
it would be better to use the size of the img array instead of a hardcoded "5" since you might get an out of bounds error if the size of the img array changes